roles/openshift_facts/library/openshift_facts.py

```diff
@@ -932,15 +934,19 @@ def set_version_facts_if_unset(facts):
             version_gte_3_4_or_1_4 = True
             version_gte_3_5_or_1_5 = True
             version_gte_3_6 = True
+            version_gte_3_7 = True
```

Comment:
This should, in fact, be `True` if we are flipping openshift-ansible to install 3.7 by default.  I thought the intent was to allow the installer to install 3.7, not set it as the new default.